src/FileHelper.php

```diff
@@ -5,9 +5,18 @@
 class FileHelper
 {
 
-	public static function isAbsolutePath(string $path): bool
+	public static function getWorkingDirectory(): string
 	{
-		return substr($path, 0, 1) === '/' || substr($path, 1, 1) === ':';
+		return getcwd();
```

Comment:
Upravil jsem to, ale jestli někde selže `getcwd()`, tak to bude už tady: https://github.com/phpstan/phpstan/blob/master/bin/phpstan#L8